hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AsyncFSWAL.java

```diff
@@ -363,7 +362,8 @@ private void syncCompleted(AsyncWriter writer, long processedTxid, long startTim
   // sync futures then just use the default one.
   private boolean isHsync(long beginTxid, long endTxid) {
```

Comment:
Just to note that above the methods are changed from getTxid to getTxId but then here we have beginTxid...  super nit... but you might want them to be consistent? Just leave the getTxid?